src/app/app.module.ts

```diff
@@ -1,19 +1,26 @@
-import { BrowserModule } from '@angular/platform-browser';
-import { NgModule } from '@angular/core';
-import { AppComponent } from './app.component';
-import { ContentComponent } from '../content/content.component';
-import { HttpClientModule } from '@angular/common/http';
-import { SearchInputComponent } from '../shared-components/search-input/search-input.component';
-import { AppMaterialModule } from './app-material.module';
-import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
-import { ButtonComponent } from '../shared-components/button/button.component';
-import { DialogComponent } from '../shared-components/dialog/dialog.component';
-import { ApiService } from '../services/api.service';
-import { FormsModule, ReactiveFormsModule } from '@angular/forms';
-import { MoviesService } from '../services/movies.service';
-import { ListItemComponent } from '../shared-components/list-item/list-item.component';
-import { CheckBoxComponent } from '../shared-components/check-box/check-box.component';
-import { PaginatorComponent } from '../shared-components/paginator/paginator.component';
+import {BrowserModule} from '@angular/platform-browser';
+import {NgModule} from '@angular/core';
+import {AppComponent} from './app.component';
+import {ContentComponent} from '../content/content.component';
+import {HttpClientModule} from '@angular/common/http';
+import {SearchInputComponent} from '@shared-components/search-input/search-input.component';
+import {AppMaterialModule} from './app-material.module';
+import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
+import {ButtonComponent} from '@shared-components/button/button.component';
+import {DialogComponent} from '@shared-components/dialog/dialog.component';
+import {ApiService} from '@services/api.service';
+import {FormsModule, ReactiveFormsModule} from '@angular/forms';
+import {MoviesService} from '@services/movies.service';
+import {ListItemComponent} from '@shared-components/list-item/list-item.component';
+import {CheckBoxComponent} from '@shared-components/check-box/check-box.component';
+import {PaginatorComponent} from '@shared-components/paginator/paginator.component';
+import {StoreModule} from '@ngrx/store';
+import {EffectsModule} from '@ngrx/effects';
+import {StoreDevtoolsModule} from '@ngrx/store-devtools';
+import {environment} from '@environments/environment';
+import {MovieEffects} from '../store/movie/movie.effects';
```

Comment:
to co możesz zrobić żeby troche wyczyscic te importy to:
- w folderze `store` mozesz stworzyc plik index.ts w którym zaimportujesz effectsy z folderów i wyeksportujesz całą tablice `export const effects = `MovieEffects`, tak samo z reducerami: `export const reducers = { data:  movieReducer.reducer, ui: uiReducer.reducer }`, dzięki temu zaimportujesz tutaj juz gotową listę effectow i reducerów